### ISSUE TITLE
`filters.gaussian` ignores `preserve_range`

### ISSUE THREAD

**Issue by stefanv** [scikit-image/scikit-image#3999]
```
from skimage import filters
import numpy as np

x = filters.gaussian(np.array([[0, 0.5, 200, 1000]]), preserve_range=True)
y = filters.gaussian(np.array([[0, 0.5, 200, 1000]]), preserve_range=False)

np.testing.assert_array_equal(x, y)
```

**Comment by sciunto** [comment]
I don't think this is a bug. Docstring says:

`        Whether to keep the original range of values. Otherwise, the input
        image is converted according to the conventions of `img_as_float`.`

https://github.com/scikit-image/scikit-image/blob/master/skimage/filters/_gaussian.py#L60

 and img_as_float says:

>     Notes
    -----
    The range of a floating point image is [0.0, 1.0] or [-1.0, 1.0] when
    converting from unsigned or signed datatypes, respectively.
    If the input image has a float type, intensity values are not modified
    and can be outside the ranges [0.0, 1.0] or [-1.0, 1.0].

**Comment by sciunto** [comment]
@stefanv Do you agree that we can close this ussue? or you think that the behavior must be changed?

**Comment by rfezzani** [comment]
I agree with @sciunto, this doesn't look like a bug but the expected result.
